### PR TITLE
echo last commit sha to the version.txt file

### PR DESCRIPTION
**Pull request by dstodolny**
* added version.txt file generation with last commit SHA while building the application

### CODE REVIEW

File: bin/ci-deploy.sh [hunk 1]
```diff
@@ -27,14 +32,19 @@ createCommitSite() {
...
   make build
+
+  echo $COMMIT_REF > ./dist/version.txt
+
   echo Copying assets to S3
   aws s3 sync ./dist/assets-honestly s3://$BUCKET_NAME/assets-honestly
   aws s3 cp ./dist/manifest.json s3://$BUCKET_NAME/manifest.json
   aws s3 cp ./dist/robots.txt s3://$BUCKET_NAME/robots.txt
   aws s3 cp ./dist/sw.js s3://$BUCKET_NAME/sw.js
+  aws s3 cp ./dist/version.txt s3://$BUCKET_NAME/$COMMIT_REF/
```
Comment (asavin) [inline bin/ci-deploy.sh]: No need for `$COMMIT_REF/` here

**Comment by gregdorward** [comment]
29c4e6b is displayed as expected when appending /version.txt to the url

**Comment by gregdorward** [comment]
This one has been tested since the previous update (see last comment), so just needs another code review to be deployed

**Comment by chris-brookes** [comment]
@asavin @dstodolny  - Can we add the build date/time details onto the page please. That info will help and make it easier when checking the release on live.

File: bin/ci-deploy.sh [hunk 2]
```diff
@@ -27,14 +33,20 @@ createCommitSite() {
 }
 
 deployMaster() {
+  COMMIT_REF=$(git rev-parse --short HEAD)
```
Comment (asavin) [inline bin/ci-deploy.sh]: you can prob skip this line now

**Pull request merged**